test/extension.test.ts

```diff
@@ -8,38 +8,36 @@ suite('Fabric8 Analytics Extension', () => {
     assert.ok(vscode.extensions.getExtension('redhat.fabric8-analytics'));
   });
 
-  test('should activate', function () {
+  test('should activate', async function () {
```

Comment:
```suggestion
  test('should activate', async () => {
```

Anyways you are adding async, change it to arrow functions

Reply:
ack